tests/MongoDB.Driver.Core.Tests/Core/Operations/FindOperationTests.cs

```diff
@@ -274,24 +321,22 @@ public void CreateFindCommandOperation_should_return_expected_result_when_modifi
                     { "$showDiskLoc", true },
                     { "$snapshot", true }
                 }
+#pragma warning restore 618
             };
 
             var result = subject.CreateFindCommandOperation();
-
+#pragma warning disable 618
```

Comment:
Preserve the blank line betwwen 327 and 328 (it's the blank line between the act/assert sections).

Reply:
Done